### PR TITLE
Google signin

### PR DESCRIPTION
**Pull request by iamtarun1993**

### CODE REVIEW

**Review by modernwarfareuplink — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,6 +23,34 @@ Note: Do not make any changes to environment.ts file - it is to be a template fo
...
  - npx cap sync
+ - npx cap copy
  - npx cap open android
  - under build of top navigation - select build variant (debug or release)
- - build apk from the same as above
\ No newline at end of file
+ - build apk from the same as above
+ ** IF npx cap give some trouble, please replace it with `ionic cap`
+
+## For running app directly in android device for staging
```
Comment (Julias0) [inline README.md:32]: #

File: README.md [hunk 2]
```diff
@@ -23,6 +23,34 @@ Note: Do not make any changes to environment.ts file - it is to be a template fo
...
+- ionic capacitor run android -l --external --configuration=staging
+  This will open android studio, let it build index file and gradle build for sometime
+  Then check that studio recgnized the right device in top bar. Press run button. After every change you make in `src` directory.
+  It will automaticallybuild the app in device.
+
+## Google Login
```
Comment (Julias0) [inline README.md:38]: #

File: src/app/auth/sign-in/sign-in.page.ts [hunk 3]
```diff
@@ -26,7 +26,8 @@ export class SignInPage implements OnInit {
     private popoverController: PopoverController,
     private loaderService: LoaderService,
     private authService: AuthService,
-    private router: Router
+    private router: Router,
+    public googleAuthService :GoogleAuthService
```
Comment (Julias0) [inline src/app/auth/sign-in/sign-in.page.ts:30]: public googleAuthService: GoogleAuthService

File: src/app/core/services/google-auth.service.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { GooglePlus } from '@ionic-native/google-plus/ngx';
+import { environment } from 'src/environments/environment';
+import { Observable } from 'rxjs';
```
Comment (Julias0) [inline src/app/core/services/google-auth.service.ts:4]: I think this is not used

File: src/app/core/services/google-auth.service.ts [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  ) { }
+
+  async login(): Promise<any>{
+    // Need to put ios case here later
+    var clientId = environment.ANDROID_CLIENT_ID;
+    return this.googlePlus.login({
+      webClientId: clientId,
+      offline: false
+    }).then(res => {
+      this.googlePlus.logout();
```
Comment (Julias0) [inline src/app/core/services/google-auth.service.ts:22]: Didnt understand this?

**Review by Julias0 — APPROVED** [review]
Minor comments + 1

File: src/app/core/services/google-auth.service.ts (hunk 5 above)
Comment (iamtarun1993) [reply src/app/core/services/google-auth.service.ts:22]: resolved offline

Comment (iamtarun1993) [reply src/app/core/services/google-auth.service.ts:22]: resolved offline

**Pull request closed without merging**